src/test/java/ru/training/at/hw7/dataProvider/ParametersForCheckingMetalAndColorsForm.java

```diff
@@ -1,6 +1,6 @@
 package ru.training.at.hw7.dataProvider;
 
-public class Data {
+public class ParametersForCheckingMetalAndColorsForm {
```

Comment:
Not a good rename: usually the last word is the meaning, so this should be Form, but is it's not a Form.
Please review name convention for Java once again and choose another name

Reply:
Improved